acquia_lift_profiles/acquia_lift_profiles.module

```diff
@@ -6,6 +6,7 @@
  */
 
 define('ACQUIA_LIFT_PROFILES_THUMBNAIL_WIDGET_SEPARATOR', '|');
+define('ACQUIA_LIFT_PROFILES_DEFAULT_IDENTITY_TYPE_DEFAULT', 'email');
```

Comment:
This was intentional. Like THUMBNAIL_WIDGET_SEPARATOR is "thumbnail widget's separator value" above, DEFAULT_IDENTITY_TYPE_DEFAULT is "default identity type's default value". "Default identity type" itself wouldn't be a constant.